src/components/Button.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ name, color, wide }) => (
-  <button className={`button ${color} ${wide ? 'large' : 'small'}`} type="button">{name}</button>
+const Button = ({
```

Comment:
**REQUIRED**
* Implements `handleClick` method that takes `buttonName` as a parameter and returns `clickHandler` method from the props.([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/button/implementHandleClick.png)).